### ISSUE TITLE
path of json file

### ISSUE THREAD

**Issue by thomasmattheussen** [Updater/node-sass-json-importer#2]
Hi,

Great work so far, this will definitely come in handy!

I was just wondering how to reference the json file in my .scss? To what does a relative path point to, and from where?

I think something is wrong with this line:

```
let file = resolve(prev.slice(0, prev.lastIndexOf('/')), url);
```

Because when I log ```prev``` I get ```"stdin"``` (which makes it search files in a non-existent stdi/ folder)

Also, I was just thinking that it might be useful to also resolve paths from node-sass' ```includePaths``` option?

**Comment by pmowrer** [comment]
Thanks for the feedback @thomasmattheussen!

I would imagine that the json file should be referenced relative to the file that's importing it. That's really the only case I've tried. If you have other use cases, please let me know and I'll look into it. You're right in that it should probably look at `includePaths` as well.

**Comment by thomasmattheussen** [comment]
Hmm, I probably wasn't very clear: I mean that from what I see, it's NOT relative to the file that's importing it... It's not that I have another use case, it's just not working as expected. 

The ```prev``` parameter that the importer function receives seems to be ```"stdin"``` in my case, and the library uses that to find the path of the json file...

Sidenote: I'm using gulp-sass in through Laravel Elixir (though I don't think that's important)

**Comment by pmowrer** [comment]
I just published `1.0.1` which should take `includePaths` into account. Please let me know if that resolves the issue.

**Comment by SpenceDiNicolantonio** [comment]
@pmowrer I just tried version 1.01. You're expecting the `includePaths` option to be an array, when the entries in `includePaths` actually get concatenated into a `:`-delimited string.

Where you have:
```Javascript
    let paths = []
        .concat(prev.slice(0, prev.lastIndexOf('/')))
        .concat(this.options.includePaths || []);
```
I think you want something more like:
```Javascript
    var includePaths = this.options.includePaths ? this.options.includePaths.split(':') : [];
    let paths = []
        .concat(prev.slice(0, prev.lastIndexOf('/')))
        .concat(this.options.includePaths || []);
```
That got things working for me.

**Comment by pmowrer** [comment]
Assuming this can be closed with the merging of #4. Thanks again @SpenceDiNicolantonio
